docker/README.md

```diff
@@ -0,0 +1,18 @@
+####Requirements
+* Docker
+* Docker-compose
+
+####Getting Started
+
+1. Rename **.env.example** to **.env**.
+2. Change `ALLOWED_HOSTS=''` to `ALLOWED_HOSTS=127.0.0.1` in **.env**
+3. Run `docker-compose up`
+4. Create super user with `docker exec -it officemanager_web_1 python3 manage.py createsuperuser`
+5. Open `http://127.0.0.1/admin/` to be sure that everything works
+
+
+####Deleting project
+If you need to delete the project (for example, to install it again) you need to do the following:
+1. Stop and remove all project containters with `docker-compose down` in project's root folder
```

Comment:
`directory` instead of `folder`. We've got used to the Unix-like operating systems=)